lib/mlcatconfig/driver_path.py

```diff
@@ -0,0 +1,24 @@
+import configparser
+import os
+
+# mailbox_list = ['lkml', 'opensuse-kernel', 'opensuse-features', 'opensuse', 'opensuse-bugs', 'opensuse-factory', 'sakai-devel']
+class Config(configparser.ConfigParser):
+    def __init__(self,mailbox):  
```

Comment:
It is better to have the following method signature:
```
    def __init__(self,mailbox,cfgfile=null): 
```
If the cfgfile is not specified, then you can use the default config file available in the mlconfig module. However, if the user specifies the config file, then the provided config file must be used.